packages/main/src/MultiComboBox.js

```diff
@@ -122,6 +122,18 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Indicates whether the dropdown is open. True if the dropdown is open, false otherwise.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		opened: {
+			type: Boolean,
+			defaultValue: false,
```

Comment:
For Boolean properties, we don't write the` defaultValue`, because it is always false (can't be never true). It's good to have it in the jsdoc, so `@defaultvalue false` is correct, just remove `defaultValue: false` from the "opened" object in the metadata.

Reply:
Thanks! I will remove it.